### PR TITLE
update copyrights in all files

### PR DESCRIPTION
**Pull request by kavithacm**
Signed-off-by: Kavitha Conjeevaram Mohan <mohakavi@amazon.com>

### Description
[Describe what this change achieves]
 
### Issues Resolved
[List any issues this PR will resolve]
 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: opensearch-notebooks/gradlew [hunk 1]
```diff
@@ -1,19 +1,8 @@
 #!/usr/bin/env sh
 
 #
-# Copyright 2015 the original author or authors.
```
Comment (joshuali925) [inline opensearch-notebooks/gradlew:4]: this is from apache license. could you keep the original license, and add our full license on top?

File: opensearch-notebooks/gradlew.bat [hunk 2]
```diff
@@ -1,17 +1,7 @@
 @rem
-@rem Copyright 2015 the original author or authors.
```
Comment (joshuali925) [inline opensearch-notebooks/gradlew.bat:2]: same as above

File: opensearch-notebooks/gradlew (hunk 1 above)
Comment (kavithacm) [reply opensearch-notebooks/gradlew:4]: yeah okay

File: opensearch-notebooks/gradlew.bat (hunk 2 above)
Comment (kavithacm) [reply opensearch-notebooks/gradlew.bat:2]: okayss

**Review by joshuali925 — APPROVED** [review]
lgtm

**Review by anirudha — APPROVED** [review]
Lgtm

**Pull request merged**